### ISSUE TITLE
Can't delete user account via 'Delete Account' link from 'Code Triage misses you' email

### ISSUE THREAD

**Issue by ahmetabdi** [codetriage/codetriage#339]
http://www.codetriage.com/users/unsubscribe/-random-letter-number-combo-here-

"We're sorry, but something went wrong."

**Comment by schneems** [comment]
Did you change your github username? I don't see you in our database

```
irb(main):007:0> User.where(github: "ahmetabdi").first
=> nil
irb(main):010:0> User.where("github ilike '%ahmetabdi%'").first
=> nil
```

**Comment by ahmetabdi** [comment]
@schneems always been the same

**Comment by chrisccerami** [comment]
I haven't looked at the code for this yet, but I suspect it's trying to render a page with data that no longer exists, since you said it looks like the account was deleted. I'll check it out and see if I can find the offending code.

**Comment by schneems** [comment]
> maybe it did delete

Probably. Have you been getting any code triage emails?

**Comment by chrisccerami** [comment]
I'm thinking what happened is @ahmetabdi tried to delete their account twice. If they tried to visit the unsubscribe url after already deleting the account, you'd get an error, because the account isn't found to be deleted. I can submit a pull request to change this so it will render an error explaining that the account wasn't found and may have already been deleted if you'd like.

**Comment by schneems** [comment]
That would be great. If someone tries to delete a user account that doesn't exist, we should either say so, or simply return "it worked!". 

Maybe we could also link to the github issues and ask people to open a ticket if they cannot stop getting emails. Also calling out a changed github username as a likely source of error would be good. There may be a better way to make our deletion code stronger. I don't know if github provides a UUID for users, but if they do we should probably switch to recording and using that for deletion.

**Comment by schneems** [comment]
Should be fixed by #342
